app/components/Header/DropdownContents/ProfileDropdown/ProfileDropdown.css

```diff
@@ -0,0 +1,42 @@
+@import url('https://fonts.googleapis.com/css2?family=Lora&display=swap');
```

Comment:
Do you really need a seperate font just for this... It just adds to the page load time with _another_ font being loaded, and for a single word.

Reply:
Probably not, and I strongly considered it, but thought I would see what you guys thought about it 😅